### PR TITLE
Update taiwaneseMandarin.xml

### PR DESCRIPTION
**Pull request by zica87**

### CODE REVIEW

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml [hunk 1]
```diff
@@ -780,7 +780,7 @@
                     <Item id="6110" name="使用中的頁籤加色條"/>
                     <Item id="6111" name="顯示狀態列"/>
                     <Item id="6112" name="在每個頁籤顯示關閉按鈕"/>
-                    <Item id="6113" name="雙擊滑鼠關閉文件"/>
+                    <Item id="6113" name="按兩下關閉文件"/>
```
Comment (donho) [inline PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:783]: is it better "滑鼠點兩下關閉文件" ?

**Review by donho — CHANGES_REQUESTED** [review]
Change from:
`<Item id="6113" name="按兩下關閉文件"/>`
to:
`<Item id="6113" name="滑鼠點兩下關閉文件"/>`

File: PowerEditor/installer/nativeLang/taiwaneseMandarin.xml (hunk 1 above)
Comment (zica87) [reply PowerEditor/installer/nativeLang/taiwaneseMandarin.xml:783]: OK.

**Comment by donho** [comment]
@zica87 
Could you complete the request please?

**Comment by donho** [comment]
Close due to no reply of PR author.

**Pull request closed without merging**